gateway-ha/src/main/java/io/trino/gateway/ha/router/OAuth2GatewayCookie.java

```diff
@@ -25,15 +25,8 @@ public class OAuth2GatewayCookie
     public static final String NAME = GatewayCookie.PREFIX + "OAUTH2";
     public static final String OAUTH2_PATH = "/oauth2";
 
-    public OAuth2GatewayCookie(String backend)
+    public OAuth2GatewayCookie(String backend, List<String> deletePaths, Duration ttl)
     {
-        super(
-                NAME,
-                null,
-                backend,
-                ImmutableList.copyOf(Streams.concat(Stream.of(OAUTH2_PATH), OAuth2GatewayCookieConfigurationPropertiesProvider.getInstance().getDeletePaths().stream()).toList()),
-                OAuth2GatewayCookieConfigurationPropertiesProvider.getInstance().getDeletePaths(),
-                OAuth2GatewayCookieConfigurationPropertiesProvider.getInstance().getLifetime(),
-                0);
+        super(NAME, null, backend, Streams.concat(Stream.of(OAUTH2_PATH), deletePaths.stream()).toList(), deletePaths, ttl, 0);
```

Comment:
No need to use `Stream` for concatenating a string and a list. Please consider using Immutable list builder.